beacon-chain/blockchain/forkchoice/process_block.go

```diff
@@ -68,7 +69,7 @@ func (s *Store) OnBlock(ctx context.Context, b *ethpb.BeaconBlock) error {
 	if err != nil {
 		return errors.Wrapf(err, "could not get signing root of block %d", b.Slot)
 	}
-	if err := s.verifyBlkDescendant(ctx, root, b.Slot); err != nil {
+	if err := s.verifyBlkDescendant(ctx, bytesutil.ToBytes32(b.ParentRoot), b.Slot); err != nil {
```

Comment:
This was right before, why did you change it to parent root?

Reply:
Discussed offline, the reason is we don't save block before (for a good reason) Looking up via parent root is should be ok : )